azure-spring-boot/src/main/java/com/microsoft/azure/keyvault/spring/KeyVaultEnvironmentPostProcessorHelper.java

```diff
@@ -81,62 +71,58 @@ public void addKeyVaultPropertySource() {
         }
     }
 
-    ServiceClientCredentials getCredentials() {
-        if (this.environment.containsProperty("MSI_ENDPOINT")
-                && this.environment.containsProperty("MSI_SECRET")) {
-            LOG.debug("Will use MSI credentials for app services");
-            final String msiEndpoint = getProperty(this.environment, "MSI_ENDPOINT");
-            final String msiSecret = getProperty(this.environment, "MSI_SECRET");
-            final AppServiceMSICredentials msiCredentials = new AppServiceMSICredentials(AzureEnvironment.AZURE,
-                    msiEndpoint, msiSecret);
-
-            if (this.environment.containsProperty(Constants.AZURE_KEYVAULT_CLIENT_ID)) {
-                LOG.debug("Will use MSI credentials for app services with user assigned identity");
-                final String clientId = getProperty(this.environment, Constants.AZURE_KEYVAULT_CLIENT_ID);
-                msiCredentials.withClientId(clientId);
-            }
-
-            return msiCredentials;
-        }
-
-        final long timeAcquiringTimeoutInSeconds = this.environment.getProperty(
-                Constants.AZURE_TOKEN_ACQUIRE_TIMEOUT_IN_SECONDS, Long.class, Constants.TOKEN_ACQUIRE_TIMEOUT_SECS);
+    TokenCredential getCredentials() {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use explicit scoping instead of the default package private level](https://app.codacy.com/manual/Microsoft/azure-spring-boot/pullRequest?prid=4796516)